### PR TITLE
#35663 - Meta viewport tag

### PR DESCRIPTION
**Pull request by kjugi**
+ Add all required meta tags

#147 

https://dev.snowdog.pro/issues/35663

### CODE REVIEW

File: components/_preview.hbs [hunk 1]
```diff
@@ -2,7 +2,13 @@
     <head>
         <meta charset="utf-8">
         <title>{{ title }}</title>
-        <link media="all" rel="stylesheet" href="{{ path '/css/styles.css' }}">
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="Alpaca components based on gulp and fractal. Part of alpaca project. Required in Alpaca M2 theme">
+        <meta name="keywords" content="components, gulp, js, scss, fractal, magento2">
```
Comment (Igloczek) [inline components/_preview.hbs]: It's not necessary, it's just the preview loaded as iframe.

File: components/_preview.hbs [hunk 2]
```diff
@@ -2,7 +2,13 @@
     <head>
         <meta charset="utf-8">
         <title>{{ title }}</title>
-        <link media="all" rel="stylesheet" href="{{ path '/css/styles.css' }}">
+        <meta charset="utf-8">
```
Comment (Igloczek) [inline components/_preview.hbs]: Duplicated, check line 2.

File: components/_preview.hbs (hunk 1 above)
Comment (kjugi) [reply components/_preview.hbs]: ok, removed

File: components/_preview.hbs (hunk 2 above)
Comment (kjugi) [reply components/_preview.hbs]: checked, removed line 2 😁

**Pull request merged**